### PR TITLE
CW-212 Match CSS spacing to design guidelines

### PR DESCRIPTION
**Pull request by JarvisWang0715**
# Change

1. Adapt the new spacing system
2. Remove the extra padding in the menu links

## Change reason

To make the current menu/spacing system align with the current wireframe.

## Comments

@glebme I have fixed the spacing system and @sergiomercado19  I removed the extra spacing in the menu links (seems like we don't need to add extra padding to make it align left or right)

### CODE REVIEW

File: frontend/src/styles/base/_spacing.scss [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
-$space-xxxl: 8.5em;
+$space-xxxs: 20px;
+$space-xxs: 40px;
+$space-xs: 60px;
+$space-sm: 80px;
+$space-md: 100px;
+$space-lg: 160px;
+$space-xl: 200px;
+$space-xxl: 240px;
+$space-xxxl: 280px;
```
Comment (glebme) [inline frontend/src/styles/base/_spacing.scss:16]: I'm worried about this when this transfers to mobile. In one sense that keeping the em around means that the spacing is proportional to the font size which will change based on the device/browser. Thoughts? You could always make space-xxxs for example 1.25em which in a desktop browser be 20pts.

**Review by glebme — DISMISSED** [review]
LGTM if you could reply to my comment, and we can go from there.

File: frontend/src/styles/base/_spacing.scss (hunk 1 above)
Comment (JarvisWang0715) [reply frontend/src/styles/base/_spacing.scss:16]: ok I will change them into em

File: frontend/src/styles/base/_spacing.scss [hunk 2]
```diff
@@ -5,15 +5,15 @@
...
-$space-xs: 0.5em;
-$space-sm: 0.75em;
-$space-md: 1.25em;
-$space-lg: 2em;
-$space-xl: 3.25em;
-$space-xxl: 5.25em;
-$space-xxxl: 8.5em;
+$space-xxxs: 1.25em; // 20px on desktop
+$space-xxs: 2.5em;   // 40px on desktop
+$space-xs: 3.75px;   // 60px on desktop
```
Comment (glebme) [inline frontend/src/styles/base/_spacing.scss:10]: Still px here? Is that on purpose?

Comment (JarvisWang0715) [reply frontend/src/styles/base/_spacing.scss:10]: Ah .... hahah nope just a typo

**Review by glebme — APPROVED** [review]

**Pull request merged**